app-metrics/node_exporter/files/node_exporter-1.6.1.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Prometheus exporter for machine metrics
+Documentation=https://github.com/prometheus/node_exporter
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+User=node_exporter
+Group=node_exporter
+Restart=on-failure
+Type=simple
+ExecStart=/usr/sbin/node_exporter
```

Comment:
In https://salsa.debian.org/go-team/packages/prometheus-node-exporter/-/blob/debian/sid/debian/service I see that they pass in $ARGS from an environment file like this:
```service
EnvironmentFile=/etc/default/prometheus-node-exporter
ExecStart=/usr/bin/prometheus-node-exporter $ARGS
```
Maybe we should do this to align functionality with openrc?
```service
EnvironmentFile=/etc/conf.d/node_exporter
ExecStart=/usr/sbin/node_exporter $command_args
```

Reply:
I think conf.d is specific to openrc stuff., dont like to use it for systemd. Users can easily create drop in file and customize with `systemctl edit node_exporter`. The whole point of systemd is to avoid such hacks. 

If you insist, I can use `systemd_install_serviced` and install 00gentoo.conf for it.

Reply:
Yeah, I suppose `systemctl edit` is convenient enough. Thanks!